ecscollector/collector.go

```diff
@@ -142,6 +174,41 @@ func (c *collector) Collect(ch chan<- prometheus.Metric) {
 		log.Printf("Failed to retrieve metadata: %v", err)
 		return
 	}
+
+	ch <- prometheus.MustNewConstMetric(
+		metadataDesc,
+		prometheus.GaugeValue,
+		1.0,
+		metadata.Cluster,
+		metadata.TaskARN,
+		metadata.Family,
+		metadata.Revision,
+		metadata.DesiredStatus,
+		metadata.KnownStatus,
+		metadata.PullStartedAt,
+		metadata.PullStoppedAt,
+		metadata.AvailabilityZone,
+		metadata.LaunchType,
+	)
+
+	svcLabelsVals := []string{
+		metadata.TaskARN,
+	}
+
+	ch <- prometheus.MustNewConstMetric(
+		svcCpuLimitDesc,
+		prometheus.GaugeValue,
+		float64(metadata.Limits.CPU),
+		svcLabelsVals...,
```

Comment:
Why are there labels on this metric (and the svcMemLimit)? We have the TaskARN on the info metric.

Reply:
fixed